features/group.feature

```diff
@@ -17,14 +17,6 @@ Feature: Group
       And I press "Create"
     Then I should see "This value is too short. It should have 3 characters or more."
 
-  Scenario: Add group validations 2
```

Comment:
I think this test can stay, only the validation of the collection is removed but not the validation for the name.

Can you also provide a more descriptive scenario: something like Add a group without a required name. So when the test fails it provides better information.